src/__tests__/components/RelatedVideoCard/RelatedVideoCard.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+
+import Adapter from 'enzyme-adapter-react-16';
+import EnzymeToJson from 'enzyme-to-json';
+import { mount, configure } from 'enzyme';
+
+import RelatedVideoCard from '../../../components/RelatedVideoCard';
+
+describe('RelatedVideoCard', () => {
+  let relatedVideoCard;
+  beforeEach(() => {
+    configure({ adapter: new Adapter() });
+    relatedVideoCard = mount(
+      <BrowserRouter>
+        <RelatedVideoCard />
+      </BrowserRouter>
+    );
+  });
```

Comment:
You could remove this shared variable `relatedVideoCard`. It is best to keep each unit test isolated. The result would be like so:
 ```
 it('Renders VideoDetails Page', () => {
    const wrapper = mount(
      <BrowserRouter>
        <RelatedVideoCard />
      </BrowserRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
```